src/main/java/org/coffeemine/app/spring/data/ITask.java

```diff
@@ -13,6 +13,10 @@
 
     void setName(@NotNull String name);
 
+    @NotNull String getAssignSprint();
```

Comment:
this reduced OOP design and its better to make a helper function in the database to search for you

Reply:
+1